dev-docs/modules/adpod.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: page_v2
+page_type: module
+title: Module - Adpod
+description: Adds functions to validate, cache, and modify long-form video bids.
+module_code : Adpod
+display_name : Adpod
+enable_download : true
+sidebarType : 1
+---
+
+# Adpod Module
+
+The adpod module provides functions to validate, cache, and modify long-form video bids. 
+
+## How to use the module:
```

Comment:
This module would only be used for developers who want to add support for a new adserver that handles long-form/adpod videos (like freewheel).

In their equivalent `<name>AdserverVideo` module, they'd just need to import the `initAdpodHooks` function and call it somewhere in their file (eg `initAdpodHooks();`)

There are other optional values they can import from the adpod module; these are:
`TARGETING_KEY_PB_CAT_DUR`
`TARGETING_KEY_CACHE_ID`

These variables are strings that contain the adpod specific targeting keys.  Currently they equate to `hb_pb_cat_dur` and `hb_cache_id` respectively.  The developer can choose to use these variables in their code to generate the correct output (ie targeting keys) to send to their adserver.

Reply:
Revised doc, please review for accuracy.